docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -15,6 +15,76 @@ ms.service: multiple
 
 # Azure CLI 2.0 release notes
 
+## January 31, 2018
+
+Version 2.0.26
+
+### Core
+
+* Added support raw token retrival in MSI context
+* Removed polling indicator string after finishing LRO on Windows cmd.exe
+* Added a warning that appears when using a configured default has been changed to an INFO level entry. Use `--verbose` to see
+* Add a progress indicator for wait commands
+
+### ACS
+
+* Clarified `--disable-browser` argument
+* Improved tab completion for `--vm-size` arguments
+
+### Appservice
+
+* Fixed `webapp log [tail|download]`
+* Removed the `kind` check on webapps and functions
+
+### CDN
+
+* Fixed missing client issue with `cdn custom-domain create`
+
+### CosmosDB
+
+* Fixed parameter description for failover policies
+
+### Interactive
+
+* Fixed issue where command option completions no longer appeared
+
+### Network
+
+* Added protection for `--cert-password` to `application-gateway create`
+* Fixed issue with `application-gateway update` where `--sku` erroneously applied a default value
+* Added protection for `--shared-key` and `--authorization-key` to `vpn-connection create`
+* Fixed missing client issue with `asg create`
+* Fixed the following issues with `dns zone export`:
+  * Added `--file-name/-f` parameter for exported names
+  * Fixed issue where long TXT records were incorrectly exported
+  * Fixed issue where quoted TXT records were incorrectly exported without escaped quotes
+* Fixed issue where certain records were imported twice with `dns zone import` 
+* Restored `vnet-gateway root-cert` and `vnet-gateway revoked-cert` commands
+
+### Profile
+
+* Fixed `get-access-token` to work inside a VM with identity
+
+### Resource
+
+* Fixed bug with `deployment [create|validate]` where warning was incorrectly displayed when a template 'type' field contained uppercase values
+
+### Storage
+
+* Changed `storage account update` to not create a new networkRuleSet if "default\_action" arg is not provided [X]
```

Comment:
This looks like an internal-only change? Is this affected by user input in any way?

Reply:
@williexu ?

Reply:
This was needed due to some service behavior. Customers who were updating their old storageV1 accounts to storageV2 accounts were encountering errors due to CLI logic.

Reply:
For change logs, try and describe the user-facing changes rather than internal details. This is a conceptual, published document for end-users. Developers who need to track internal changes have `git log` and `git blame`.